cumulusci/cli/tests/test_cci.py

```diff
@@ -224,6 +225,7 @@ def test_project_init(self, click):
             run_click_command(cci.project_init, obj=config)
 
             # Make sure expected files/dirs were created
+            print recursive_list_files()
```

Comment:
use python3 friendly print fn plz

Reply:
Heh, this was just meant for temporary debugging anyway. Will remove.